src/main/java/htsjdk/variant/variantcontext/writer/VariantContextWriterBuilder.java

```diff
@@ -107,6 +104,7 @@
     public static final EnumSet<Options> DEFAULT_OPTIONS = EnumSet.of(Options.INDEX_ON_THE_FLY);
     public static final EnumSet<Options> NO_OPTIONS = EnumSet.noneOf(Options.class);
     private static final OpenOption[] EMPTY_OPEN_OPTION_ARRAY = new OpenOption[0];
+    private final static Log log = Log.getInstance(VariantContextWriter.class);
```

Comment:
incorrect modifier order
```suggestion
    private static final Log log = Log.getInstance(VariantContextWriter.class);
```